src/models/payment.js

```diff
@@ -0,0 +1,14 @@
+const mongoose = require("mongoose");
+const { Schema } = mongoose;
+
+const schema = new Schema({
+  concept: String,
+  amount: { type: Number, required: true },
+  paymentStatus: { type: String, required: true },
+  date: { type: Date, required: true, default: Date.now },
+  user: { type: [Schema.Types.ObjectId], ref: "User" },
```

Comment:
Schema.Types.ObjectId can be replaced to mongoose.ObjectId - I'm guessing you want to save the ID of the user

Reply:
and is it better?